### PR TITLE
fix: Issues running some make tasks on windows Fixes #7604

### LINKED ISSUE #7604: Issues running some make tasks on windows
Some make tasks don't run properly on Windows due to use of commands like "rm".

Proposing to update the dev-environment-quick-start-guide.md with details on how to get around this.

### PR DESCRIPTION
**Pull request by john-gom**
### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

File: docs/introduction/dev-environment-quick-start-guide.md [hunk 1]
```diff
@@ -205,3 +205,24 @@ You need to remove current directory where you clone the project, and clone the
...
 git clone -c core.symlinks=true git@github.com:openfoodfacts/openfoodfacts-server.git
+```
+
+### 'rm' is not recognized as an internal or external command
+
+When running make import_prod_data or some other commands.
+
+Solution:
+
+Use the Git Bash shell to run the make commands in windows so that programs like nproc and rm are found.
```
[the remark this answers is not in the document]
Comment (alexgarel) [reply docs/introduction/dev-environment-quick-start-guide.md:216]: (in the prerequisite sections, a sub-section for windows)

**Review by alexgarel — APPROVED** [review]
Cool, thanks.
I made a suggestion to add a prerequisit section for windows, would you like to add it to this PR (or make a new PR ?)

File: docs/introduction/dev-environment-quick-start-guide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (john-gom) [reply docs/introduction/dev-environment-quick-start-guide.md:230]: I added a specific Windows prerequisites section. Is this the sort of thing you were looking for?

File: docs/introduction/dev-environment-quick-start-guide.md [hunk 2]
```diff
@@ -20,7 +20,21 @@ Docker provides an isolated environment, very close to a Virtual Machine. This e
 > If you run e.g. Debian, don't forget to add your user to the `docker` group!
 - [Install Docker Compose](https://docs.docker.com/compose/install/)
 - [Enable command-line completion](https://docs.docker.com/compose/completion/)
-- [Install Make for Windows](http://gnuwin32.sourceforge.net/packages/make.htm) (if running on Windows)
+
+### Windows Prerequisites
+
+When running with Windows, install [Docker Desktop](https://www.docker.com/products/docker-desktop/) which will cover all of the above.
+
+The Make tasks use a number of Linux commands, such as rm and nproc, so it is recommeded to run Make commands from the Git Bash shell. In addition, the following need to be installed and inclided in the PATH:
```
Comment (stephanegigandet) [inline docs/introduction/dev-environment-quick-start-guide.md:28]:
```suggestion
The Make tasks use a number of Linux commands, such as rm and nproc, so it is recommeded to run Make commands from the Git Bash shell. In addition, the following need to be installed and included in the PATH:
```

**Review by stephanegigandet — APPROVED** [review]
Looks good, thank you!

**Pull request merged**